src/views/Nft/market/Collection/IndividualNFTPage/PancakeBunnyPage/ForSaleTableCard/index.tsx

```diff
@@ -114,27 +153,23 @@ const ForSaleTableCard: React.FC<ForSaleTableCardProps> = ({ bunnyId, nftMetadat
             <PageButtons>
               <Arrow
                 onClick={() => {
-                  if (page !== 1) {
-                    setPage(page - 1)
-                  }
+                  switchPage(internalPage === 1 ? internalPage : internalPage - 1)
                 }}
               >
-                <ArrowBackIcon color={page === 1 ? 'textDisabled' : 'primary'} />
+                <ArrowBackIcon color={internalPage === 1 ? 'textDisabled' : 'primary'} />
               </Arrow>
-              <Text>{t('Page %page%', { page })}</Text>
+              <Text>{t('Page %page% of %maxPage%', { page: internalPage, maxPage: maxInternalPage })}</Text>
```

Comment:
I think `maxPage` shouldn't be displayed because the number isn't the maximum page which might make users confused.

Reply:
Done